src/patternfly/demos/Toolbar/examples/toolbar-attribute-value-single-select-filter-example.hbs

```diff
@@ -0,0 +1,75 @@
+{{#> data-toolbar data-toolbar--id="data-toolbar-attribute-value-single-select-filter-example"}}
+  {{#> data-toolbar-content}}
+    {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-toggle-group pf-m-reveal-on-lg" data-toolbar-group--IsExpanded="false"}}
+      {{> data-toolbar-toggle}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-filter-group"}}
+      {{#> data-toolbar-item}}
+        {{#> select id="select-two" select-toggle--icon="fas fa-filter"}}
+          Status
+        {{/select}}
+      {{/data-toolbar-item}}
+      {{#> data-toolbar-item}}
+        {{#> select select--IsExpanded="true" id="select-three"}}
+          Stopped
+        {{/select}}
+      {{/data-toolbar-item}}
+      {{/data-toolbar-group}}
+    {{/data-toolbar-group}}
+    {{#> overflow-menu overflow-menu--modifier="pf-m-show-on-xl" overflow-menu--id="overflow-menu-simple-example" overflow-menu-button--aria-label="Generic options" overflow-menu-dropdown-button-aria-expanded="aria-expanded=true"}}
+  {{#> overflow-menu-content}}
+    {{#> overflow-menu-item}}
+      {{#> data-toolbar-group data-toolbar-group--modifier="pf-m-button-group"}}
+        {{#> data-toolbar-item}}
+          {{#> button button--modifier="pf-m-primary"}}
+            Action
+          {{/button}}
+        {{/data-toolbar-item}}
+        {{#> data-toolbar-item}}
+          {{#> button button--modifier="pf-m-link"}}
+            Action
+          {{/button}}
+        {{/data-toolbar-item}}
+        {{#> data-toolbar-item}}
+          {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Actions"'}}
+            <i class="fas fa-ellipsis-v" aria-hidden="true"></i>
+          {{/button}}
+        {{/data-toolbar-item}}
+      {{/data-toolbar-group}}
+      {{/overflow-menu-item}}
+      {{/overflow-menu-content}}
+  {{#> overflow-menu-control}}
+    {{#> overflow-menu-dropdown-item}}
+      Action
+    {{/overflow-menu-dropdown-item}}
+    {{#> overflow-menu-dropdown-item overflow-menu-dropdown-item--IsShared="true"}}
+      Item 1
+    {{/overflow-menu-dropdown-item}}
+    {{#> overflow-menu-dropdown-item overflow-menu-dropdown-item--IsShared="true"}}
+      Item 2
+    {{/overflow-menu-dropdown-item}}
+  {{/overflow-menu-control}}
+{{/overflow-menu}}
+
+
+    {{#> data-toolbar-item data-toolbar-item--modifier=(concat 'pf-m-pagination ' data-toolbar-item-pagination--modifier)}}
+      {{#> pagination pagination--modifier="pf-m-compact"}}
+        {{#> pagination-total-items}}
+          <b>1 - 10</b> of <b>37</b>
+        {{/pagination-total-items}}
+        {{> pagination-options-menu options-menu id="pagination-options-menu-compact-example"  options-menu--IsText="true"}}
```

Comment:
I added this issue https://github.com/patternfly/patternfly-next/issues/2290 but you can go ahead and remove `options-menu` and `id="pagination-options-menu-compact-example"` from this (and the other) examples. Doesn't look like they're used.

Reply:
ok thanks for that, removed